### PR TITLE
Update Known-issues.md

### PR DESCRIPTION
**Pull request by RuiPereiraTabares**
|**Issue title**|**Behavior / Symptom**|**Known workaround**|**Discovery date**|
|:-----|:-----|:-----|:-----|
|Teams Planner integration with Planner online <br/> |Owners are unable to create a plan from a team created from a existing office 365 group.  <br/> |Give the menber permissions to the group owner. <br/> |1/14/20  <br/>|

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.
--

**Comment by kenwith** [comment]
Closing this as I had an expert research and it doesn't appear to be a bug. See note below.

This doesn’t qualify as a “Known Issue/Bug”, it’s likely that the user just had permissions to the associated SharePoint site in some other fashion via SharePoint permissions/sharing links.

**Comment by kenwith** [comment]
Closed by mistake. The comment above should have been for https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/3773

File: Teams/Known-issues.md [hunk 1]
```diff
@@ -308,6 +308,14 @@ This article lists the known issues for Microsoft Teams, by feature area.
 |:-----|:-----|:-----|:-----|
 |Teams Planner integration with Planner online <br/> |Tasks buckets in Planner do not show up in Planner online experience.  <br/> |No workaround. <br/> |2/28/17  <br/>|
 
+
+|**Issue title**|**Behavior / Symptom**|**Known workaround**|**Discovery date**|
+|:-----|:-----|:-----|:-----|
+|Teams Planner integration with Planner online <br/> |Owners are unable to create a plan from a team created from a existing office 365 group.  <br/> |Give the menber permissions to the group owner. <br/> |1/14/20  <br/>|
```
Comment (JohanFreelancer9) [inline Teams/Known-issues.md]:
```suggestion
|Teams Planner integration with Planner online <br/> |Owners are unable to create a plan for a team created from an existing Office 365 group.  <br/> |Give member permissions to the group owner. <br/> |1/14/20  <br/>|
```

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by RuiPereiraTabares** [comment]
@LolaJacobsen  can you take a look?

**Comment by rjagiewich** [comment]
Per @kenwith it's OK to merge this one.

**Comment by LolaJacobsen** [comment]
@marclauinger Please review.

**Pull request merged**